<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project X ver. 0.0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Otwór</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Wartości</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>A11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>R6</string>
        </property>
       </item>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>181.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>47</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>111</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>xxx</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Wałek</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Wartości</string>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>c11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>j6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>js4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>js12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>js16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>r6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>r7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s7</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>47</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>xxx</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Przeliczenia</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Cale&lt;-&gt;milimetry</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Cale na mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mm na cale</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Przelicz</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wyczyść</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>269</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionInformacje"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInformacje">
   <property name="text">
    <string>Informacje</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
